src/browser/provider/built-in/base.js

```diff
@@ -41,7 +41,7 @@ export default class BrowserProviderBase {
         this.resizeCorrections[id] = { width: newWidth - width, height: newHeight - height };
     }
 
-    async resizeWindow (id, pageInfo, width, height) {
+    async resizeWindow (id, width, height, pageSize) {
```

Comment:
`pageSize` is a current client area's width and height (`window.innerWidth` and `window.innerHeight`) before resize, `width`and `height` are desired width and height of client area after resize.